larray/tests/test_array.py

```diff
@@ -176,40 +174,53 @@ def small_array(meta):
 io_narrow_missing_values[2, 'b1', 'c1'] = np.nan
 
 
-def test_ndrange():
-    arr = ndtest('a=a0..a2')
+def test_concat(meta):
```

Comment:
the name of the function is misleading... it seems to be general while it only tests metadata.